src/app/api/auth/[auth0]/route.ts

```diff
@@ -12,4 +12,9 @@ export const GET = auth0.handleAuth({
   "oidc-logout": auth0.handleLogout({
     returnTo: oidcLogoutUrl.toString(),
   }),
+  "silent-login": auth0.handleLogin({
```

Comment:
Sends the user back through login without displaying any UI. This is used to update the user's id and access tokens with any new claims following onboarding. Right now all this adds is the `owa:onboarded: true` claim to their id token which will show up in their user object. If they're not already logged in this will error.